jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java

```diff
@@ -460,7 +460,7 @@ protected Request copyRequest(HttpRequest oldRequest, URI newURI)
                 continue;
 
             String value = field.getValue();
-            if (!newRequest.getHeaders().contains(header, value))
+            if (!newRequest.getHeaders().contains(field.getName(), value))
                 newRequest.header(field.getName(), value);
```

Comment:
Can you please refactor `field.getName()` into local variable `name` _before_ the declaration of local variable `value` ?